i18n/en.json

```diff
@@ -14,6 +14,12 @@
 	"echo-pref-tooltip-requestssl-request-comment": "Notify me when there is a comment on an SSL certificate request that I have participated on.",
 	"echo-pref-tooltip-requestssl-request-status-update": "Notify me when the status is updated for an SSL certificate request that I have participated on.",
 	"requestssl-button-copy": "(Copy to Clipboard)",
+	"requestssl-cloudflare-comment-active": "This custom domain has been successfully activated and applied to this wiki.",
```

Comment:
Suggestion, not requirement:
`This custom domain has been successfully activated and is now applied to this wiki.`